src/components/TermGroup.js

```diff
@@ -4,6 +4,9 @@ import lexiconTerms from '../../content/lexicon.json';
 
 const TermGroup = ({ value, onChange }) => {
     // Set default value - cleaner way possible?
+    if (typeof(value) === "undefined") {
+        value = {};
+    }
```

Comment:
```suggestion
    value = value || {};
```

should work (not tested, in which case is this needed?)

Reply:
Great! This was needed because this happened to occur in some cases, leading to crashes...